packages/react/src/components/sideNavigation/SideNavigation.stories.tsx

```diff
@@ -48,7 +48,7 @@ export const Default = (args) => {
       </style>
       <div className="example-page">
         <div className="example-page-side-navigation">
-          <SideNavigation {...args}>
+          <SideNavigation ariaLabel="Side Navigation" {...args}>
```

Comment:
`ariaLabel="Side Navigation"`

This can possibly be confusing because the name originates from the visual position of the navigation element. I will confirm this from accessibility people.

Reply:
I changed it to "Components pages navigation" and "About pages navigation", etc.